pom.xml

```diff
@@ -237,7 +237,6 @@
                             <url>https://github.com/ERDDAP/erddapContent/releases/download/${erddapcontent.download.version}/erddapContent.zip</url>
                             <unpack>true</unpack>
                             <outputDirectory>${project.basedir}</outputDirectory>
-                            <cacheDirectory>${project.basedir}/download_cache</cacheDirectory>
```

Comment:
Won't removing the cache directory cause the files to need to be downloaded repeatedly?

Reply:
Nope, removing `cacheDirectory` will cause the default of `~/.m2/repository/.cache/maven-download-plugin` to be used for the cache, and since that's inside of `~/.m2/repository` it will be included in the GitHub action maven cache.

https://github.com/maven-download-plugin/maven-download-plugin/blob/0d0fd979794a5b0c941ea5175a891cc8f22c432a/src/main/java/com/googlecode/download/maven/plugin/internal/WGetMojo.java#L191

https://github.com/actions/setup-java/blob/main/src/cache.ts#L26